src/main/java/io/seqera/tower/cli/utils/FormatHelper.java

```diff
@@ -334,4 +334,19 @@ public static String formatLabels(java.util.List<io.seqera.tower.model.LabelDbDt
         .collect(Collectors.joining(","));
     }
 
+    public static String formatDescription(String description) {
+        return formatDescription(description, 2048);
+    }
+
+    public static String formatDescription(String description, int maxLength) {
+        if (description == null) {
+            return "NA";
+        }
+
+        // remove line breaks
+        var text = description.replace("\n", " ").replace("\r", " ");
```

Comment:
very minor: maybe we could remove leading and trailing spaces as well with trim
```suggestion
        var text = description.trim().replace("\n", " ").replace("\r", " ");
```

Reply:
Yep, valid point